parachains/runtimes/starters/seedling/src/lib.rs

```diff
@@ -14,6 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
+//! # Seedling Runtime
+//!
+//! Seedling is a parachain meant to help parachain auction winners migrate a blockchain from
+//! another consensus system (including its own) into the consensus system of a given Relay Chain.
```

Comment:
This "including its own" confuses me. What you want to express with this?

Reply:
That it could be a parachain of another Relay Chain or a standalone chain. But yeah, the meaning stands without it so I'll remove it.